rel-eng/tito.props

```diff
@@ -23,6 +23,7 @@ whitelist = foreman-bootloaders-redhat
   foreman-release
   foreman-release-scl
   foreman-selinux
+  nodejs-angular
```

Comment:
Shouldn't these be in the katello-nightly-rhel7 whitelist?

Reply:
Probably why the build failed